site/docs/plugins/menu.md

```diff
@@ -5,7 +5,8 @@ Easily create interactive menus.
 ## Introduction
 
 An inline keyboard is an array of buttons underneath a message.
-grammY has [a built-in plugin](./keyboard.md#inline-keyboards) to create basic inline keybaords.
+grammY has [a built-in plugin](./keyboard.md#inline-keyboards)
+to create basic inline keyboards.
```

Comment:
Please revert this change. The docs are sticking with a convention of having one sentence per line.
```suggestion
grammY has [a built-in plugin](./keyboard.md#inline-keyboards) to create basic inline keyboards.
```

Reply:
However, I can't understand the reasoning behind this, considering that it's becoming extremely hard to work with the file, cause it's not fitting on the screen, but it's your monastery :)